cmd/pulumi-test-language/interface.go

```diff
@@ -730,199 +730,231 @@ func (eng *languageTestServer) RunLanguageTest(
 		}
 	}
 
-	// Create a source directory for the test
-	sourceDir := filepath.Join(token.TemporaryDirectory, "source", req.Test)
-	err = os.MkdirAll(sourceDir, 0o700)
-	if err != nil {
-		return nil, fmt.Errorf("create source dir: %w", err)
-	}
-
-	// Find and copy the tests PCL code to the source dir
-	err = copyDirectory(languageTestdata, filepath.Join("testdata", req.Test), sourceDir)
-	if err != nil {
-		return nil, fmt.Errorf("copy source test data: %w", err)
-	}
+	var result LResult
+	for i, run := range test.runs {
```

Comment:
This multi-run behaviour needs to be tested, e.g. see l2resourcesimple_test.go. We'll want a l2destroy_test.go.

Reply:
Added a test :heavy_check_mark: